module/VuFind/src/VuFind/RateLimiter/RateLimiterManagerFactory.php

```diff
@@ -135,6 +135,9 @@ protected function getRateLimiter(
         }
 
         $rateLimiterConfig = $policy[$configSection] ?? [];
+        if ('reject_all' == $rateLimiterConfig['policy'] ?? null) {
```

Comment:
Doesn't null coalescing have lower precedence than comparison? If so, you'll need parenthesis. I'd also use strict comparison:
```suggestion
        if ('reject_all' === ($rateLimiterConfig['policy'] ?? null)) {
```